packages/e2e-tests/specs/block-grouping.test.js

```diff
@@ -173,4 +175,37 @@ describe( 'Block Grouping', () => {
 			expect( await getEditedPostContent() ).toMatchSnapshot();
 		} );
 	} );
+
+	describe( 'Registering alternative Blocks to handle Grouping interactions', () => {
```

Comment:
This is exactly as I imagined when we discussed the possible solution on Slack (links requires registration):

https://wordpress.slack.com/archives/C02QB2JS7/p1561454889265000

Reply:
That's good news!